docs/blog/write-the-docs-gets-you-a-job.rst

```diff
@@ -0,0 +1,80 @@
+Write the Docs gets you a job
+=============================
+
+Write the Docs is a community for everyone who cares about documentation.
+Over time,
+we have brought together people from myriad companies and job titles to share in their passion for communication and helping users.
+Our in person and online spaces provide a wonderful place to learn best practices around documentation.
+Because of this,
+we have attracted a large number of people at the forefront in thinking around documentation.
+These best practices continue to push the industry forward,
+shared through our conferences and online spaces.
+
+Along with pushing the industry forward,
+we know that each person in our community is also trying to push their career forward.
+People in our community want to work with forward thinking companies.
+They want coworkers who *get it*,
+who care about their customers as much as they do.
+We have been seeing this happen organically in our *#jobs* channel in Slack for years now,
+and now we're launching two efforts to connect great folks with great companies.
+
+The Job Board
+-------------
+
+We are launching a new Job Board: http://jobs.writethedocs.org.
```

Comment:
We are launching a new Job Board http://jobs.writethedocs.org that has two simple functions:

* Employers can post open positions
* Candidates can search for jobs and set up alerts